src/transformers/models/dpt/modeling_dpt.py

```diff
@@ -1036,6 +1074,10 @@ def forward(self, hidden_states: List[torch.Tensor]) -> torch.Tensor:
         # use last features
         hidden_states = hidden_states[self.config.head_in_index]
 
+        if self.projection is not None:
+            hidden_states = self.projection(hidden_states)
+            hidden_states = ACT2FN["relu"](hidden_states)
```

Comment:
```suggestion
            hidden_states = nn.ReLu(hidden_states)
```
No need to use ACT2FN here no?